repository-data/webfiles/src/main/resources/site/freemarker/common/facets.ftl

```diff
@@ -1,7 +1,7 @@
 <#ftl output_format="HTML">
 <#include "../include/imports.ftl">
 <!--Need to have a single setBundle call as subsequent ones will overwrite the previous values-->
-<@hst.setBundle basename="document-types,month-names,publicationsystem.labels,facet-headers"/>
+<@hst.setBundle basename="document-types,month-names,publicationsystem.labels,nationalindicatorlibrary.labels,facet-headers,facet-labels"/>
```

Comment:
no need for `nationalindicatorlibrary.labels` and I think `publicationsystem.labels` are no longer used as well